test/unittests/libtesteth/blockchainTest.cpp

```diff
@@ -18,11 +18,133 @@
  * Unit tests for blockchain test filling/execution.
  */
 
-#include <boost/test/unit_test.hpp>
 #include <test/tools/libtesteth/TestOutputHelper.h>
+#include <test/tools/libtesteth/TestHelper.h>
+
+#include <boost/test/unit_test.hpp>
 
 using namespace dev::test;
 
 BOOST_FIXTURE_TEST_SUITE(BlockChainTestSuite, TestOutputHelper)
 
+BOOST_AUTO_TEST_CASE(fillingExpectationOnMultipleNetworks)
+{
+	std::string const s = R"(
+		{
+			"BLOCKHASH_Bounds" : {
```

Comment:
I think we could define this template once and then use a replace function from fuzzTest.cpp
you could define your own map for replacement. even something like 
[EXPECT] -> "{ "network" : "Frontier", ...  }"